dataprofiler/tests/profilers/test_profiler_options.py

```diff
@@ -359,4 +357,21 @@ def test_data_labeler(self, *mocks):
                                     load_options=None)
         actual_sample_size = profile._profile[0].profiles['data_label_profile'] \
             ._profiles["data_labeler"]._max_sample_size
-        self.assertEqual(actual_sample_size, 50)
\ No newline at end of file
+        self.assertEqual(actual_sample_size, 50)
```

Comment:
the following adds the test for the checking if data labeler can be set properly via the options